Gordon360/Services/NewsService.cs

```diff
@@ -25,26 +35,41 @@ public NewsService(IUnitOfWork unitOfWork)
         /// controller
         /// </summary>
         /// <param name="newsID">The SNID (id of news item)</param>
+        /// <param name="needsImageData">Whether or not the place from which
+        /// Get was invoked needed to get the image path or image base64 data. For
+        /// calls from editPosting, path is needed. For calls from uploadImage,
+        /// path is needed. But for calls from the controller (frontend), the data is needed.</param>
```

Comment:
This is a pretty clunky way to handle this requirement. In my mind, it would be much more natural for this method to return the News item as it is in the database (including the path to the image if it exists). If the image data is needed, the place where it is needed (such as the `GET` route) could then call a `GetImage` method passing in the image path as retrieved from this method.

Reply:
I can get right to it!